admin/javascript/src/components/grid-field/row.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class GridFieldRowComponent extends SilverStripeComponent {
+
+  render() {
+    return (
+      <li className="grid-field-row-component [ list-group-item ]">{this.props.children}</li>
```

Comment:
@scott1702 I assume you're referring to the small width table in https://projects.invisionapp.com/boards/9GX68PTXUQFH#/1547737/73357185? I think that's still a table actually even on the mobile breakpoints - it just exposes another piece of content ("20 changes") underneath the title. What requires a lot of JavaScript here?